### ISSUE TITLE
Checkpoint 1

### ISSUE THREAD

**Issue by jeliceiri** [jeliceiri/MyPlacesTracker#1]
@pawaitemadisoncollege - checkpoint 1 is ready for review

Completed:
- Problem statement
- User stories
- Project plan
- Screen design 
- TimeLog
- Pushed to GitHub
- Stories identified that are part of the Minimum Viable Product (MVP)

**Comment by pawaitemadisoncollege** [comment]
Hi @jeliceiri - you are off to a great start with MyPlacesTracker! Your problem statement, solution and list of tech look very good. Screen design and user stories seem well-thought-out. Looking at the stories marked for MVP hits the project requirements and feels like a well-sized project for the semester. I'm looking forward to seeing this built out!

Project plan - starts off strong with good detail in the first weeks and then... well, it could use more detail. I know it can be hard to identify everything that needs to be done, so here are a few suggestions:

- [x] It might be helpful to consider which user stories you will work on in a given week.  Will you have all jsps and classes built in week 4? It might be helpful to iterate a bit, building jsps and classes for a single story and then moving on to the next story. This way you can incorporate what you learn in future stories.  
- [x] You have several apis listed, will you write the code to consume all of them in a single week? Week 8?
- [x] CSS framework?  Might be good to add a task for selecting and implementing.

**Comment by jeliceiri** [comment]
@pawaitemadisoncollege 

Thanks for the feedback Paula! 

I have been working on incorporating your suggestions into the project plan. I also clarified which API's will be part of what I plan to build in the README.md (hospital capacity - which will depend on zip code lookup and county fips codes) and which are for a future version (adding weather and google location). I also added a task to select and implement CSS framework.
